my-vault/handlers/handlers.go

```diff
@@ -0,0 +1,164 @@
+package handlers
+
+import (
+	"encoding/json"
+	"fmt"
+	"net/http"
+	"github.com/google/uuid"
+	"sync"
+)
+
+type Secret struct {
+	Id    string 
+	Name  string  
+	KVMap map[string]string 
```

Comment:
What do you think about making this a bit simpler for now and just store a `Value string`?